lib/rouge/lexers/hack.rb

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    load_lexer 'php.rb'
+
+    class Hack < PHP
+      title 'Hack'
+      desc 'The Hack programming language (hacklang.org)'
+      tag 'hack'
+      aliases 'hack', 'hh'
+      filenames '*.php', '*.hh'
+
+      def self.analyze_text(text)
```

Comment:
Filetype guesser is changed in https://github.com/jneen/rouge/pull/763

Can you take a look at it?